src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php

```diff
@@ -38,27 +37,24 @@ class DebugHandlersListener implements EventSubscriberInterface
     private $levels;
     private $throwAt;
     private $scream;
-    private $fileLinkFormat;
     private $scope;
     private $firstCall = true;
     private $hasTerminatedWithException;
 
     /**
-     * @param callable|null                 $exceptionHandler A handler that must support \Throwable instances that will be called on Exception
-     * @param array|int                     $levels           An array map of E_* to LogLevel::* or an integer bit field of E_* constants
-     * @param int|null                      $throwAt          Thrown errors in a bit field of E_* constants, or null to keep the current value
-     * @param bool                          $scream           Enables/disables screaming mode, where even silenced errors are logged
-     * @param string|FileLinkFormatter|null $fileLinkFormat   The format for links to source files
-     * @param bool                          $scope            Enables/disables scoping mode
+     * @param callable|null $exceptionHandler A handler that must support \Throwable instances that will be called on Exception
+     * @param array|int     $levels           An array map of E_* to LogLevel::* or an integer bit field of E_* constants
+     * @param int|null      $throwAt          Thrown errors in a bit field of E_* constants, or null to keep the current value
+     * @param bool          $scream           Enables/disables screaming mode, where even silenced errors are logged
+     * @param bool          $scope            Enables/disables scoping mode
      */
-    public function __construct(callable $exceptionHandler = null, LoggerInterface $logger = null, $levels = \E_ALL, ?int $throwAt = \E_ALL, bool $scream = true, $fileLinkFormat = null, bool $scope = true)
+    public function __construct(callable $exceptionHandler = null, LoggerInterface $logger = null, $levels = \E_ALL, ?int $throwAt = \E_ALL, bool $scream = true, bool $scope = true)
```

Comment:
that's a BC break!

Reply:
can we tagging it as `@internal` and just remove it from 5.x? I think this class isn't intended to be used in userland.

Reply:
@nicolas-grekas the class is marked a final, so it's not a BC break?!

Reply:
it's a BC break for ppl that instantiate this class directly, final or not
we cannot remove this argument from 5.x, but we can deprecate it yes, and remove it in v6

Reply:
Ah, yes 👍🏼 
I'll do that

Reply:
Done 👍🏼